keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -1022,11 +1022,11 @@ class SqlDeliveryConfigRepository(
                 .execute()
             }
           }
-          .map { (_, name) ->
-            get(name)
-          }
       }
     }
+      .map { (_, name) ->
+        get(name)
+      }
```

Comment:
This is the fix -- once we have the candidates we want to load, do so _after_ the transaction has completed, otherwise the write to `DELIVERY_CONFIG_LAST_CHECKED.AT` will be rolled back if the read fails.